### DISCUSSION TITLE
Project level redirects

### DISCUSSION THREAD

**Discussion by kasperaamodt** [vercel/community#2688] [Help]
### Problem Description

Hello,

Is it possible to have project level redirects? My use case is that I have one git repo that I deploy in multiple projects, that connects to different backends with the help of env variables. So I can't put my redirects in vercel.json or next config, because those files are shared between all the projects.

**Comment by amyegan** [comment]
You can use [Edge Middleware](https://vercel.com/docs/concepts/functions/edge-middleware) with [Edge Config](https://vercel.com/docs/storage/edge-config) to customize redirects per project. 

There are quite a few [Edge Middleware templates](https://vercel.com/templates?type=edge-middleware) available, but I know the [Maintenance Page template](https://vercel.com/templates/next.js/maintenance-page) shows how to use Edge Config with Middleware redirects. You can find the example Middleware here: https://github.com/vercel/examples/blob/main/edge-middleware/maintenance-page/middleware.ts
